tests/feature/test_steps.py

```diff
@@ -489,19 +495,27 @@ def test_step_trace(pytester):
         from pytest_bdd import given, when, scenario
 
         @given('I have a bar')
-        def _():
+        def i_have_bar():
```

Comment:
Why? I want to encourage users to define steps functions using underscore, since they are not supposed to be invoked directly, and since the name is actually given by the `name` Parma in the decorator